packages/language-support/src/autocompletion/completionCoreCompletions.ts

```diff
@@ -32,7 +32,7 @@ import { CompletionItem, Neo4jFunction, Neo4jProcedure } from '../types';
 const uniq = <T>(arr: T[]) => Array.from(new Set(arr));
 
 function backtickIfNeeded(e: string): string | undefined {
-  if (/\s/.test(e) || /[^a-zA-Z]/.test(e)) {
+  if (/\s/.test(e) || /[^a-zA-Z0-9_]/.test(e)) {
```

Comment:
There's an edge case we are not really handling well here: a label, rel type or property key that starts with a number still needs backticks

Reply:
Also checked with chatGPT and apparently non-english letters is okay anywhere in the names, will fix both at the same time

Reply:
I've just checked with an instance of the database to be sure. Please check always with the database itself because you've seen ChatGPT already give hallucinations regarding neo4j documentation for procedures.

Reply:
I do indeed know about hallucinations and have also checked, no need to worry